packages/node/src/node/integration/ProtocolService.ts

```diff
@@ -502,8 +502,20 @@ function clusterTypeProtocolOf(backing: BehaviorBacking): ClusterTypeProtocol |
                 const {
                     access: { limits },
                 } = behavior.supervisor.get(member);
+                const {
+                    effectiveQuality: { largeMessage },
+                } = member;
+
+                const command = {
+                    id: id as CommandId,
+                    responseId,
+                    requestTlv,
+                    responseTlv,
+                    limits,
+                    name,
+                    largeMessage: largeMessage ? largeMessage : undefined,
```

Comment:
I think this should go into limits instead.  Timed is there too

Reply:
But that would mix quality and access then, really sure?

Reply:
Yes I think that's ok